frontend/components/TableContainer/DataTable/ActionButton/ActionButton.tsx

```diff
@@ -67,7 +93,8 @@ ActionButton.propTypes = {
   targetIds: PropTypes.arrayOf(PropTypes.number),
```

Comment:
I believe you can remove the `propTypes` object since the interface is being used.

Reply:
Deleting `propTypes` seems to create a linting error, e.g., '`targetIds' is missing in props validation (eslintreact/prop-types)`. How would you like to resolve?